loader/loader.go

```diff
@@ -680,9 +682,12 @@ func (l *Loader) prepareDbFiles(files map[string]struct{}) error {
 		}
 	}
 
-	if len(l.db2Tables) == 0 {
+	if schemaFileCount == 0 {
 		return errors.New("invalid mydumper files for there are no `-schema-create.sql` files found")
 	}
+	if len(l.db2Tables) == 0 {
+		return errors.New("no available `-schema-create.sql` files, check mydumper parameters and bwlist config")
```

Comment:
use name in _task.yaml_ rather than `bwlist`?

Reply:
changed to `no available `-schema-create.sql` files, check mydumper parameter matches black-white-list in task config`